esic_fe/src/components/Navbar/Navbar.jsx

```diff
@@ -1,12 +1,57 @@
-import React from "react";
+import React, { useState } from "react";
 import { Link } from "react-router-dom";
 
 export function Navbar() {
-    return (
-        <nav className="text-secondary-900">
-        <ul>
-            Navbar
-        </ul>
+    
+    const [navbar,setnavbar] = useState(false);
```

Comment:
```suggestion
    const [navbar, setNavbar] = useState(false);
```

You could have used better variable names, use verbs instead is my suggestion

it can be const [open, isOpen] = useState (false), if you tend to refer the opening and closing of the hamburger

and an industry standard practice is to generally follow camelCase convention:

ex: const [navbar, setNavbar] = useState(false)

Reply:
okay I'll keep that in mind